crypto_utils/tests/run_test.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
```

Comment:
Is the test directory and the test files still valid ? If not, please remove it from this PR. Lot of directories referred here are not present in this repo